Scripts/RoomGenerator.gd

```diff
@@ -178,13 +178,7 @@ func connect_rooms():
 			check_connection.push_back(closest_con)
 			spanning_tree.push_back(Delaunay.Edge.new(closest_pair, closest_point))
 		else: break
-	
-	var con_count = 0
-	for c in connection:
-		con_count += c.connected_room.size()
-	
-	var keep_connection_count = floor(con_count * keep_connection)
-	
```

Comment:
It's been so long that I already forgot what I write in here, any reason deleting this code?

Reply:
just the editor that complains about the variable never being used